addons/mail/static/tests/message/message.test.js

```diff
@@ -130,6 +130,15 @@ test("Can edit message comment in chatter", async () => {
     await insertText(".o-mail-Message .o-mail-Composer-input", "edited message", { replace: true });
     await click(".o-mail-Message a", { text: "save" });
     await contains(".o-mail-Message-content", { text: "edited message (edited)" });
+    await click(".o-mail-Message [title='Expand']");
+    await click(".o-mail-Message-moreMenu [title='Edit']");
+    await contains(".o-mail-Message:contains('Escape to cancel, CTRL-Enter to save')");
+    await insertText(".o-mail-Message .o-mail-Composer-input", "edited again", { replace: true });
+    await press("Enter");
+    await animationFrame();
+    await contains(".o-mail-Message .o-mail-Composer-input"); // still editing message
+    await triggerHotkey("control+Enter"); // somehow press doesn't work :(
```

Comment:
should check there is a new line inside composer
especially if "press" doesn't work, why shouldn't triggerHotkey be tested instead above?

Reply:
done :)